avaframe/ana1Tests/analysisTools.py

```diff
@@ -0,0 +1,41 @@
+# imports
+import numpy as np
+
+# local imports
+import avaframe.com1DFA.DFAtools as DFAtls
+
+
+def normL2Vect(analyticalSol, numericalSol, cellSize, cosAngle):
+    """ Compute relativ L2 and Lmax norm of the error between the analytic and numerical solution
+    """
+    nonZeroIndex = np.where((np.abs(analyticalSol['fx']) > 0) & (np.abs(analyticalSol['fy']) > 0)
+                            & (np.abs(analyticalSol['fz']) > 0) & (np.abs(numericalSol['fz']) > 0)
+                            & (np.abs(numericalSol['fz']) > 0) & (np.abs(numericalSol['fz']) > 0))
+    dvx = analyticalSol['fx'] - numericalSol['fx']
+    dvy = analyticalSol['fy'] - numericalSol['fy']
+    dvz = analyticalSol['fz'] - numericalSol['fz']
+    dv = DFAtls.norm2(dvx, dvy, dvz)
+    vAnalyticalSol = DFAtls.norm2(analyticalSol['fx'], analyticalSol['fy'], analyticalSol['fz'])
+    localError = dv[nonZeroIndex]
+    vAnalyticalSolL2 = cellSize * cellSize / cosAngle * np.nansum(vAnalyticalSol[nonZeroIndex])
+    vAnalyticalSolL2 = np.sqrt(vAnalyticalSolL2)
+    error2 = cellSize * cellSize / cosAngle * np.nansum(localError)
+    error = np.sqrt(error2) / vAnalyticalSolL2
+    errorMax = np.sqrt(np.nanmax(np.append(localError, 0))) / np.sqrt(np.nanmax(np.append(vAnalyticalSol[nonZeroIndex], 0)))
```

Comment:
same code as in normLScal, why? -> either make subfunction or make it work for scalars and vectors.